clients/benchmarks/client_extop_layernorm.cpp

```diff
@@ -173,13 +214,36 @@ void compare(const char* title, const std::vector<T>& cpuOutput, const std::vect
     std::cout << "max error : " << maxErr << std::endl;
 }
 
+template <typename DType>
+void initData(DType* data, std::size_t numElements, hipblaslt_initialization initMethod)
+{
+    switch(initMethod)
+    {
+    case hipblaslt_initialization::rand_int:
+        hipblaslt_init<DType>(data, numElements, 1, 1);
```

Comment:
minus quest : should lda value be numElements?

lda == M. is it correct ?
template <typename T>
inline void hipblaslt_init(
    std::vector<T>& A, size_t M, size_t N, size_t lda, size_t stride = 0, size_t batch_count = 1)
{
    hipblaslt_init(A.data(), M, N, lda, stride, batch_count);
}